mp-istio-sample/README.md

```diff
@@ -0,0 +1,115 @@
+## MicroProfile Istio Sample
+This project is intended to show the interaction between Istio and the various MicroProfile specifications.
+
+### Requirements
+* [Git](https://git-scm.com/)
+* [Docker](https://www.docker.com/)
+* [Maven](https://maven.apache.org/install.html)
+* [Java 8]: Any compliant JDK should work.
+  * [Java 8 JDK from Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+  * [Java 8 JDK from IBM (AIX, Linux, z/OS, IBM i)](http://www.ibm.com/developerworks/java/jdk/),
+    or [Download a Liberty server package](https://developer.ibm.com/assets/wasdev/#filter/assetTypeFilters=PRODUCT)
+    that contains the IBM JDK (Windows, Linux)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+    * If your cluster is [minikube](https://kubernetes.io/docs/getting-started-guides/minikube/), you will already have this
+* [Istio](https://istio.io/)
+
+### Project Contents
+The project comprises a set of communicating services designed to be run as docker containers deployed into an Istio enabled Kubernetes cluster.
+
+Currently, the services are serviceA and serviceB which are independent maven projects each in its own subdirectory of this project.
+
+serviceA is currently the entry point for the sample. Once installed, the sample can be accessed at:
+```
+http://<your kubernetes cluster>/mp-istio-sample/serviceA
+```
+
+### Build
+
+The steps involved in building the project are:
+1.  [Build the services](#build-the-services)
+2.  [Package the services as docker images](#package-the-services)
+3.  [Push the images to a repository](#push-the-services)
+4.  [Install Istio into your cluster](#install-istio)
+5.  [Configure your cluster to run the sample](#install-the-sample-services)
+
+### Build the Services
+
+Each service resides in its own directory under this project. Currently they are serviceA and serviceB. To build the services:
+1.  (cd serviceA; mvn install)
+2.  (cd serviceB; mvn install)
+
+### Package the Services
+
+The services sould be packaged within the environment you use to access docker in your develpoment environment. Each service has a Dockerfile which is used to assemble its docker image. Each image should be tagged such that it can be subsequently pushed to your docker repository. Assuming you will be using dockerhub, and your docker id is `<docker id>`:
+1.  (cd serviceA; docker build -t `<docker id>`/mp-istio-sample-servicea .)
+2.  (cd serviceB; docker build -t `<docker id>`/mp-istio-sample-serviceb .)
+
+### Push the Services
+
+The service images need to be published to a repository so they will be available to run in your cluster. Assuming you will be using Docker Hub:
+1.  docker login
+2.  docker push 
+3.  docker push `<docker id>`/mp-istio-sample-serviceb
+
+### Install Istio
```

Comment:
Isn't installing Istio more of a pre-req than a "Build Step"?

Maybe move this bit higher up.